src/main/java/edu/illinois/cs/dt/tools/detection/detectors/ExecutingDetector.java

```diff
@@ -97,10 +97,10 @@ public void writeTo(final Path dir) throws IOException {
     }
 
     private class RunnerIterator implements Iterator<DependentTest> {
-        private final long origStartTimeMs = System.currentTimeMillis();
-        private long startTimeMs = System.currentTimeMillis();
-        private long previousStopTimeMs = System.currentTimeMillis();
-	private boolean roundsAreTotal = Boolean.parseBoolean(Configuration.config().getProperty("dt.detector.roundsemantics.total", "false"));
+    private final long origStartTimeMs = System.currentTimeMillis();
```

Comment:
These fields should be indented one more, as they are fields belonging to the RunnerIterator class.

Reply:
Yea just noticed that. Done